decidim/decidim
# Show users own voting activity

#### :tophat: What? Why?
Add "private-only" visibility option for action log entries. Private only activities are shown to user in their own activity feed. Users can only see their own "private-only" actions. For now, only budget checkout creates "private-only" action log entry.

#### :pushpin: Related Issues
https://meta.decidim.org/processes/roadmap/f/122/proposals/16765

#### Testing
1. Vote on a budget
2. Go to your activity feed
3. Logout and sign in as different user
4. Go to check activity feed of previous user

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
![image](https://user-images.githubusercontent.com/19709320/155565447-1e4118cc-bf91-47ac-b142-f1f8f78eb7f1.png)

:hearts: Thank you!
